test/integration/node-specific/examples/project_fields_from_query_results.test.js

```diff
@@ -219,4 +219,45 @@ describe('examples(project-fields-from-query):', function () {
       });
     }
   });
+
+  it('Aggregation Projection Example 1', {
+    metadata: { requires: { topology: ['single'], mongodb: '>= 4.4.0' } },
```

Comment:
Any reason to not run this on more topologies?

Reply:
All other tests on this tests file also use a single topology, and they all live on the same docs page. I'm running evergreen patch to see which topologies support this functionality; and I'll get back to you once that's finished running.

Here's the [EVG Link](https://spruce.mongodb.com/version/65f496e252556600074d1da0/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) (for reference).